### ISSUE TITLE
hspec-meta-2.3.1: QuickCheck.hs: error: Not in scope: type constructor or class ‘HasCallStack’

### ISSUE THREAD

**Issue by juhp** [hspec/hspec#284]
When building for Stackage Nightly:

```
src/Test/Hspec/QuickCheck.hs:25:10: error:
    Not in scope: type constructor or class ‘HasCallStack’
```

**Comment by sol** [comment]
Please build against the latest version of hspec-expectations

**Comment by sol** [comment]
I added a lower bound on Hackage. Anything else required from my side?

**Comment by juhp** [comment]
Thanks, that sounds fine - then I guess this will be resolved with fpco/stackage#1971 then.
